src/modules/blockchainApplication/explore/components/BlockchainApplicationDetails/BlockchainApplicationDetails.js

```diff
@@ -12,33 +12,43 @@ import Tooltip from 'src/theme/Tooltip';
 import { parseSearchParams, removeThenAppendSearchParamsToUrl } from 'src/utils/searchParams';
 import { useApplicationManagement } from 'src/modules/blockchainApplication/manage/hooks';
 import { usePinBlockchainApplication } from '@blockchainApplication/manage/hooks/usePinBlockchainApplication';
+import { useBlockchainApplicationExplore } from '../../hooks/queries/useBlockchainApplicationExplore';
+import { useBlockchainApplicationMeta } from '../../../manage/hooks/queries/useBlockchainApplicationMeta';
 import defaultBackgroundImage from '../../../../../../setup/react/assets/images/default-chain-background.png';
 import liskLogo from '../../../../../../setup/react/assets/images/LISK.png';
 import styles from './BlockchainApplicationDetails.css';
 
 const deposit = 5e10;
-const serviceUrl = 'https://lisk.com/';
 const backgroundImage = null;
-const chainLogo = null;
 
 // eslint-disable-next-line max-statements
-const BlockchainApplicationDetails = ({ history, location, application }) => {
+const BlockchainApplicationDetails = ({ history, location }) => {
   const { t } = useTranslation();
   const chainId = parseSearchParams(location.search).chainId;
   const mode = parseSearchParams(location.search).mode;
+  const { data: onChainData } = useBlockchainApplicationExplore({
+    config: { params: { chainID: chainId } },
+  });
+  const { data: offChainData } = useBlockchainApplicationMeta({
+    config: { params: { chainID: chainId } },
```

Comment:
meta data needs send network as query params,

Reply:
@soroushm The custom query already does that internally so I didn't see the need of adding that here